site/content/docs/5.0/migration.md

```diff
@@ -7,7 +7,20 @@ aliases: "/migration/"
 toc: true
 ---
 
-## Browser support
+## Alpha 2 changes
+
+While in alpha, we may introduce breaking changes as we experiment and refine some of the features.
+
+### Breaking
+
+- Renamed `.sr-only` and `.sr-only-focusable` to `.v-hidden` and `.v-hidden-focusable`.
+- Renamed `sr-only()` and `sr-only-focusable()` mixins to `v-hidden()` and `v-hidden-focusable()`.
+
+## Summary
+
+Here are the big ticket items you’ll want to be aware of when moving from v4 to v5.
```

Comment:
This should be the first thing shown at the top. Also, these will conflict with #31122 which I want to land first.

Reply:
> This should be the first thing shown at the top

i was following the format that was used here https://getbootstrap.com/docs/4.5/migration/ unless i misunderstood it...

Reply:
Better wait for #311222 and I''ll take care of this,